#!/usr/bin/env python
PACKAGE = "franka_example_controllers"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("translational_stiffness", double_t,    0, "translational stiffness", 50,  0, 1000)
gen.add("rotational_stiffness", double_t,    0, "rotational stiffness", 1,  0, 500)
gen.add("translational_damping", double_t,    0, "translational damping", 14,  0, 500)
gen.add("rotational_damping", double_t,    0, "rotational damping", 2,  0, 20)

exit(gen.generate(PACKAGE, "dynamic_compliance", "compliance_param"))
